test-suite-serde/build.gradle.kts

```diff
@@ -7,4 +7,5 @@ description = "Test suite for Liquibase + Jackson"
 dependencies {
     testImplementation(mnSerde.micronaut.serde.api)
     testImplementation(mnSerde.micronaut.serde.jackson)
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
```

Comment:
why are we still defining it here if we definitely it in test-suite-module.gradle
```suggestion
```